lib/v2/screens/explore_screens/vote_screens/proposals/viewmodels/proposals_list_event.dart

```diff
@@ -20,3 +20,9 @@ class LoadProposalsByScroll extends ProposalsListEvent {
   @override
   String toString() => 'LoadProposalsByScroll';
 }
+
+class LoadProposalsByRefresh extends ProposalsListEvent {
```

Comment:
this event is telling the bloc what to, its telling the bloc to load something. 

The job of event is to say something happened, and its up to the bloc to decide what to do. 

`OnUserProposalsRefresh`